service/src/test/java/com/epam/esm/data_provider/GiftCertificateProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.esm.data_provider;
+
+import com.epam.esm.impl.GiftCertificate;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GiftCertificateProvider {
+
+    private final TagProvider tagProvider = new TagProvider();
+
+    public List<GiftCertificate> getEmptyList() {
```

Comment:
no need for such methods - just create an empty list inline

Reply:
Fixed